binstar_client/commands/download.py

```diff
@@ -32,7 +32,7 @@ def add_parser(subparsers):
 
     parser.add_argument(
         'handle',
-        help='user/notebook',
+        help='user/package_name',
```

Comment:
maybe write this as `<channel_name>/<package_name>` like the docstring above?

Reply:
I've updated this.